subseas_flex_fcst/maproom.py

```diff
@@ -120,6 +120,16 @@ def targetStartOptions(startDate):
         optionsDict.update({leadsKeys[idx]:dateRange})
     return optionsDict, leadsKeys[0]
 
+@APP.callback(
+   Output("map_title","children"),
+   Input("variable","value"),
+   Input("startDate","value"),
+   Input("leadTime","value"),
+   Input("leadTime","options"),
+)
+def write_map_title(variable, startDate, leadTime, leadTimeOptions):
+    targetPeriod = leadTimeOptions.get(leadTime)
+    return f"{targetPeriod} {variable} Forecast issued {startDate}"
```

Comment:
For "variable," I meant to use what's in the config file which is "Precipitation Anomaly" in that case.